### ISSUE TITLE
remove "blacklist" from "Passing options" instructions for browserify

### ISSUE THREAD

**Issue by timdiggins** [babel/babel.github.io#754]
![welcome-to-open-source-atlanta](https://raw.githubusercontent.com/welcome-to-open-source/Atlanta/master/assets/banner.png)

**This issue is reserved for participants of [Welcome to Open Source, Atlanta](http://www.meetup.com/Jr-Dev-Mentoring/events/228951389/)**.

If it’s still available after April 2nd, anyone can take it.

### :grey_question: Issue

http://babeljs.io/docs/setup/#browserify includes instructions for "Passing options" which gives the sample of blacklist. (A picture of this page is shown below)

<img width="561" alt="screen shot 2016-03-29 at 10 17 04 pm" src="https://cloud.githubusercontent.com/assets/588473/14129862/046de60a-f5fc-11e5-9068-3fa43bac7cf1.png">

However the `blacklist` and other options have been removed in Babel 6 (using the current information will cause an error). So we either need to remove the "Passing options" section altogether, or replace with a valid option in Babel 6.

The file corresponding to `http://babeljs.io/docs/setup/#browserify` on the website is [here](https://github.com/babel/babel.github.io/blob/master/_includes/tools/browserify/usage.md) (_includes/tools/browserify/usage.md).

If you are logged in, you should be able to see a pencil icon on the top right of the picture below, which once clicked will bring you to an editor page where you can create a change.

<img width="1001" alt="screen shot 2016-03-29 at 10 19 16 pm" src="https://cloud.githubusercontent.com/assets/588473/14129890/4fe3e2a6-f5fc-11e5-88be-9fc627f38f32.png">

### :dart: Goal
It's better to leave in the information but with a new option.
Remove the usage of the old option `blacklist` and replace it with something else. The list of options in Babel 6 are at http://babeljs.io/docs/usage/options/. As a suggestion, I think `comments: false` could be a viable option to replace blacklist with. (FYI: Passing in this option will make babel remove all comments in the output/transpiled code)

### :clipboard: Step by Step
- [x] claim this issue (comment below)
- [x] Setup repository locally https://github.com/babel/babel.github.io#setup or edit from the github editor on [this page](https://github.com/babel/babel.github.io/blob/master/_includes/tools/browserify/usage.md) or more specifically [this](https://github.com/babel/babel.github.io/edit/master/_includes/tools/browserify/usage.md) if you are logged in.
- [x] Edit the `usage.md` file. Replace the usage of `blacklist: regenerator` with something like `comments: false`.
- [x] Commit the change. We don't have particular commit convention, but doing something like `Docs: replaced usage of old babel 5 with new one in browserify usage doc` would work.
- [x] Start a Pull Request
 (If this is your first, welcome :tada: :smile: [Here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request)
- [x] Done (Comment below to ask for a review) - Then we can give it a :+1: and merge.

---

#### Original Issue below

http://babeljs.io/docs/setup/#browserify includes instructions for "Passing options" which gives the sample of blacklist, e.g. 

```
$ browserify -d -e script.js -t [ babelify --blacklist regenerator ]
```

However blacklist has been removed. So either need to remove the "Passing options" section altogether, or replace with a valid option.

**Comment by hzoo** [comment]
@timdiggins You cool if I modify this issue for https://github.com/welcome-to-open-source/atlanta/issues/2? It's to let newcomers get into open source (or you can go ahead and submit a PR yourself and i'l find some more)

**Comment by timdiggins** [comment]
@hzoo yes, please go ahead, great idea!

**Comment by marisalaneous** [comment]
claiming this

**Comment by marisalaneous** [comment]
pr ready for review

**Comment by hzoo** [comment]
Awesome! thanks @marisalaneous

**Comment by thejameskyle** [comment]
Fixed by #777

**Comment by Leonetienne** [comment]
OP is just mentally ill
